src/components/DeckBuilderDryRunView/index.js

```diff
@@ -435,6 +465,19 @@ class DeckBuilderDryRunView extends React.Component {
                             : 'turns'}
                           .
                         </p>
+                        <p>
+                          {this.props.deck
+                            .map(card => card.id === 'W9')
+                            .filter(Boolean).length
+                            ? this.getFrozenCoreText()
+                            : null}
+                          <br />
```

Comment:
Small detail but this `<br />` will always be rendered, regardless of whether there are Frozen Core, Dawnsparks or whatnot. Maybe we remove it and use 2 different paragraphs instead?

Reply:
This would introduce a blank line between the two lines which looks odd. I will add the <br /> to the frozen core paragraph